modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/macros/EVPOrganization.macro

```diff
@@ -128,4 +128,12 @@ definition {
 			locator1 = "EVPUtils#KEBAB_MENU_OPTION_ORGANIZATION");
 	}
 
+	macro viewSuccessMessageOrganization {
+		while (IsElementPresent(locator1 = "Message#EMPTY_INFO")) {
+			AssertTextEquals(
+				locator1 = "Message#EMPTY_INFO",
+				value1 = "Thank you. Your information was successfully received.");
+		}
+	}
+
 }
```

Comment:
You can remove this macro, because is equals to the other macro, that will be replaced to EVPUtils